app/src/organisms/ProtocolSetupModules/index.tsx

```diff
@@ -120,14 +115,23 @@ function RowModule({
           </StyledText>
         </Flex>
       ) : (
-        <Chip
-          text={
-            isModuleReady ? t('module_connected') : t('module_disconnected')
-          }
-          type={isModuleReady ? 'success' : 'warning'}
-          background={false}
-          iconName="connection-status"
-        />
+        <Flex
+          flex="3 0 0"
+          alignItems={ALIGN_CENTER}
+          justifyContent={JUSTIFY_SPACE_BETWEEN}
+        >
+          <Chip
+            text={
+              isModuleReady ? t('module_connected') : t('module_disconnected')
+            }
+            type={isModuleReady ? 'success' : 'warning'}
+            background={false}
+            iconName="connection-status"
+          />
+          {isDuplicateModuleModel ? (
+            <Icon name="information" size={SIZE_2} />
```

Comment:
```suggestion
            <Icon name="information" size="2rem" />
```

conv
https://opentrons.slack.com/archives/CSCLVUW3C/p1685644862441539